ppb/events.py

```diff
@@ -262,9 +270,12 @@ class SceneStopped:
     This is delivered to a scene and it's objects when a :class:`StopScene` or
     :class:`ReplaceScene` event is sent to the engine.
 
-    The end of the scene lifetime, started with :class:`SceneStarted`.
+    Is technically an optional scene, as not all scenes in the stack will
```

Comment:
That feels weirdly worded.

Reply:
Should be `This is technically an optional event`. Will fix.